grakel_replace/mixed_single_task_gp_usage_example.py

```diff
@@ -124,8 +156,10 @@
 # Define bounds
 bounds = torch.tensor(
     [
-        [0.0] * N_NUMERICAL + [0.0] * N_CATEGORICAL,
-        [1.0] * N_NUMERICAL + [float(N_CATEGORICAL_VALUES_PER_CATEGORY - 1)] * N_CATEGORICAL
+        [0.0] * N_NUMERICAL + [0.0] * N_CATEGORICAL + [-1.0] * N_GRAPH,
+        [1.0] * N_NUMERICAL
+        + [float(N_CATEGORICAL_VALUES_PER_CATEGORY - 1)] * N_CATEGORICAL
+        + [len(X) - 1] * N_GRAPH,
```

Comment:
Because the `bounds` needs to be of the correct size, we extend it. You'll notice that I put `[-1.0]` as the lower bound. I don't like this, but it let me quickly hack in that we can use the index `-1` to refer to the last graph in the `graph_lookup`, you'll see it used later.